indico/modules/events/registration/controllers/management/description.py

```diff
@@ -0,0 +1,47 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2025 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from flask import flash
+from wtforms.fields import TextAreaField
+
+from indico.modules.events.registration.controllers.management import RHManageRegFormsBase
+from indico.modules.events.settings import EventSettingsProxy
+from indico.util.i18n import _
+from indico.web.forms.base import FormDefaults, IndicoForm
+from indico.web.forms.widgets import TinyMCEWidget
+from indico.web.util import jsonify_data, jsonify_template
+
+
+event_settings = EventSettingsProxy('registration', defaults={
+    'registration_description': ''
+})
+
+
+class RegistrationDescriptionForm(IndicoForm):
+    description = TextAreaField('Registration Description',
+                                widget=TinyMCEWidget(absolute_urls=True),
```

Comment:
I would prefer markdown here instead of full WYSIWYG HTML.

Reply:
Ok. 
@OmeGak I hope this will be fine with our needs.